ts/components/ui/Markdown/rules/link.ts

```diff
@@ -0,0 +1,26 @@
+import { Text } from "native-base";
+import * as React from "react";
+import { Linking } from "react-native";
+import { ReactOutput, SingleASTNode, State } from "simple-markdown";
+
+import { makeReactNativeRule } from ".";
+
+function rule() {
+  return (
+    node: SingleASTNode,
+    output: ReactOutput,
+    state: State
+  ): React.ReactNode => {
+    return React.createElement(
+      Text,
+      {
+        key: state.key,
+        link: true,
+        onPress: () => Linking.openURL(node.target).catch(_ => undefined)
```

Comment:
it would be nice to have a whitelist of URLs/regexes associated to the service that sends this message, but perhaps in a different story :) - cc @gunzip

Reply:
definitely, I add a story in the notification API project